docs/src/tutorials/tutorial_DegreeTwoReduction.md

```diff
@@ -0,0 +1,173 @@
+# DegreeTwoReduction
+
+In this tutorial the `DegreeTwoReduction` network reduction algorithm is presented. This reduction eliminates buses with exactly two connections by combining the incident branches into a single equivalent branch while preserving the electrical characteristics of the network.
+
+Before diving into this tutorial we encourage the user to load `PowerNetworkMatrices`, hit the `?` key in the REPL terminal and look for the documentation of `DegreeTwoReduction`.
+
+## Understanding Degree-Two Buses
+
+Degree-two buses are nodes in the network topology that have exactly two connections. These intermediate buses can be eliminated by replacing the two incident branches with a single equivalent branch, simplifying the network while maintaining its electrical behavior.
+
+## Basic Usage of DegreeTwoReduction
+
+The `DegreeTwoReduction` can be applied when constructing various network matrices. The most common use case is with the `Ybus` matrix:
+
+``` @repl tutorial_DegreeTwoReduction
+using PowerNetworkMatrices
+using PowerSystemCaseBuilder
+
+const PNM = PowerNetworkMatrices
+const PSB = PowerSystemCaseBuilder
+
+# Load a test system
+sys = PSB.build_system(PSB.PSITestSystems, "c_sys14");
+
+# Create Ybus with degree-two reduction
+ybus = Ybus(sys; network_reductions=[DegreeTwoReduction()]);
+```
+
+## Accessing Reduction Information
+
+After applying the reduction, you can access information about which buses were eliminated and how branches were combined:
+
+``` @repl tutorial_DegreeTwoReduction
+# Get the network reduction data
+reduction_data = get_network_reduction_data(ybus);
+
+# View the series branch mapping
```

Comment:
Don't suggest the use of field access as it breaks encapsulation rules required for the development of tools in the organization. If there are missing getters write the corresponding code. Dot access is actively considered accessing private information